domain/src/main/java/org/fao/geonet/domain/MetadataResourceExternalManagementProperties.java

```diff
@@ -32,21 +32,52 @@
 @XmlRootElement(name = "metadataResourceExternalManagementProperties")
 @XmlAccessorType(XmlAccessType.FIELD)
 public class MetadataResourceExternalManagementProperties {
-        private final String id;
-        private final String url;
+    private final String id;
+    private final String url;
+    private ValidationStatus status=ValidationStatus.UNKNOWN;
 
-        public MetadataResourceExternalManagementProperties(String id, String url) {
-            this.id = id;
-            this.url = url;
-        }
+    public MetadataResourceExternalManagementProperties(String id, String url) {
+        this.id = id;
+        this.url = url;
+    }
 
-        public String getUrl() {
+    public String getUrl() {
             return url;
         }
 
-        public String getId() {
+    public String getId() {
             return id;
         }
+
+    public ValidationStatus getStatus() {
+        return status;
+    }
+
+    public void setStatus(ValidationStatus status) {
+        this.status = status;
+    }
+
+    public static enum ValidationStatus {
+        UNKNOWN(0),VALID(1), INCOMPLETE(2), WARNING(3);
```

Comment:
WARNING is a nice to have feature as I'm assuming it would mean that a validation is valid but has some warning.  But that would mean that anywhere we check for VALID we would also have to check for WARNING because both are VALID. 
I don't see any usage for WARNINGS so I think this was for future use.  So I would recommend removing it for now.

Reply:
This WARNING just a place holder status and can be removed.